com.unity.netcode.gameobjects/CHANGELOG.md

```diff
@@ -1,5 +1,251 @@
+- [d6b2a486] (2021-07-29) Noel Stephens / refactor!: NetworkPrefabHandler and INetworkPrefabInstanceHandler API & XmlDoc changes (#977)
+- [4b900723] (2021-07-29) Benoit Doyon / fix: Fix named message stream copy to (#987)
+- [c25821d2] (2021-07-27) Jaedyn Draper / fix: Fixes for a few things discovered from the message ordering refactor: (#985)
+- [4fad5915] (2021-07-27) Phil Deschain / fix: Network animator server authority fixes (#972)
+- [82e1c33e] (2021-07-26) Luke Stampfli / fix: Remove assert which is currently broken (#984)
+- [b9ffc1f1] (2021-07-23) Jaedyn Draper / feat: Message Ordering (#948)
+- [fb1555f3] (2021-07-21) Luke Stampfli / feat!: Network Time (RFC #14) (#845)
+- [9ae52deb] (2021-07-21) Jeffrey Rainy / test: small fix to ManualNetworkVariableTest.cs, exposed during anoth… (#968)
+- [fa2be6f1] (2021-07-19) Jeffrey Rainy / feat: snapshot. Adding RTT computation API. (#963)
+- [bdd7d714] (2021-07-14) Jeffrey Rainy / feat: snapshot. Milestone 1b. Testproject "manual test" "scene transitioning" working with snapshot. Disabled by default. (#862)
+- [13e2b7f1] (2021-07-13) Sam Bellomo / test: multiprocess tests part 6: fixing issues runnings all tests together (#957)
+- [bf296660] (2021-07-13) Sam Bellomo / docs: Perf tests part 5. Adding documentation and instructions (#952)
+- [6c8efd66] (2021-07-12) Sam Bellomo / test: Perf tests part 4. Adding example of performance test with spawning x network objects at once (#925)
+- [089c2065] (2021-07-12) Luke Stampfli / test: Correctly teardown OnNetworkSpawn/Despawn tests.
+- [725a77a9] (2021-07-12) Sam Bellomo / test: Perf tests part 3. Adding ExecuteStepInContext for better test readability (#924)
+- [833f1faf] (2021-07-09) Sam Bellomo / test: Perf tests part 2. Adding Test Coordinator and base test class (#923)
+- [b2209c2c] (2021-07-08) Jeffrey Rainy / fix: (MLAPI.Serialization) 'specified cast is not valid.' on NetworkW… (#951)
+- [d08b84ac] (2021-07-08) Sam Bellomo / test: Perf tests part 1. Basis for multiprocess tests process orchestration.  (#922)
+- [be0ca068] (2021-07-06) Phil Deschain / feat: network animator Trigger parameter support (#872)
+- [7ed627c6] (2021-06-30) Sam Bellomo / fix: reducing log level for noisy log and adding details for developer log (#926)
+- [4679474b] (2021-06-30) Sam Bellomo / feat: users can set authority on network transform programmatically (#868)
+- [e122376f] (2021-06-29) Sam Bellomo / refactor: move NetworkBehaviour update to a separate non-static class (#917)
+- [0855557e] (2021-06-29) Sam Bellomo / test: add utils for multi instance tests (#914)
+- [9a47c661] (2021-06-29) Sam Bellomo / test: downgrading testproject to 2020.3.12f1 (#927)
+- [fae7e8ce] (2021-06-29) Noel Stephens / refactor: decouple PendingSoftSyncObjects from NetworkSpawnManager (#913)
+- [27112250] (2021-06-29) Josie Messa / chore: Change function signature of OnDespawnObject to accept NetworkObject (#928)
```

Comment:
should we have a changelog entry for this? @josiemessa 👀

Reply:
I don't think so!